intelmq/lib/bot.py

```diff
@@ -286,8 +289,10 @@ def acknowledge_message(self):
         self.__last_message = self.__current_message
         self.__source_pipeline.acknowledge()
 
-    def __dump_message(self, error_traceback):
-        if self.__current_message is None:
+    def _dump_message(self, error_traceback, message=None):
+        if message is None:
```

Comment:
I agree... if current_message lost it's meaning for the ParserBot, I would just drop it everywhere you can. The code becomes cleaner.